backend/wmg/cube_creation/loader.py

```diff
@@ -55,6 +56,11 @@ def load_h5ad(h5ad, group_name, validate):
         return
 
     anndata_object = anndata.read_h5ad(h5ad)
+
+    # Apply a low expression gene cell filtering. 
+    # Will get rid of all cells that have less than 500 expressed genes
+    scanpy.pp.filter_cells(anndata_object, min_genes=500)
```

Comment:
Should we put this number in a constant?

Reply:
I think that would be good for clarity

Reply:
Do we have a good candidate file for these constants? Or this file works?